api/server.js

```diff
@@ -1,10 +1,13 @@
 import express from 'express';
 import googleMapsClient from '@google/maps';
 import mongoose from './db/db.js';
+import dotenv from 'dotenv';
 
 import riddle from './test-routes/riddle.js';
 import gameGeneration from './routes/generateNewGame.js';
 
+dotenv.config();
```

Comment:
This is what I was talking about with the `.env` importing - I initialise it here but it doesn't extend to `db.js`